test/e2e/framework.go

```diff
@@ -1291,7 +1299,7 @@ func (data *TestData) waitForClickHousePod() error {
 	startUpdating := false
 	// ClickHouse Operator takes around 1 minute to restart the ClickHouse Pod
 	// which requires more time to make sure that ClickHouse Pod is ready.
-	err := wait.Poll(defaultInterval, defaultTimeout*2, func() (bool, error) {
+	err := wait.Poll(defaultInterval, defaultTimeout*4, func() (bool, error) {
```

Comment:
nit: update the comment in the previous line `ClickHouse Operator takes around 1 minute` since we double the timeout